languages/en.ini

```diff
@@ -382,6 +382,9 @@ fav_list_delete_fail = "Sorry, an error has occurred. Your list was not deleted.
 Favorites = "Saved Items"
 Fee = "Fee"
 Feedback = "Feedback"
+feedback_email = "Email"
+feedback_email_subject = "VuFind feedback"
```

Comment:
Do we really need to translate the feedback email subject? The email is coming to the maintainer of VuFind, not the end user. Thus, it seems that you would want a uniform subject heading that is meaningful to the maintainer and that could be filtered on, not a subject that is translated into the end user's language.

Reply:
You're right, the subject should not be translated. Possible variables within the subject are still translated.